muesli/web/templates/Fragments/main.pt

```diff
@@ -99,6 +99,10 @@ queries: ${request.queries}
 </div>
 
 </div>
+
+<!-- TODO I need this script to be executed when the page is fully loaded. If there's a better location for that, I'm happy to move it. -->
+<script tal:attributes="src request.static_url('muesli.web:static/js/accidental-action-prevention.js')"></script>
```

Comment:
You could use the event DOMContentLoaded to trigger a function, that loads the additional js file. I think this would be the way to go here

Reply:
I'm sorry, I don't quite understand how I'd do this. Where should I put the event handler? As `on...` attribute within the HTML? I have to read the docs first, I'm afraid, because I've never done that before.

Reply:
Okay, I figured it out now. Ready to merge :)